docs/docs_skeleton/docusaurus.config.js

```diff
@@ -84,7 +84,7 @@ const config = {
       ({
         docs: {
           sidebarPath: require.resolve("./sidebars.js"),
-          editUrl: "https://github.com/hwchase17/langchain/edit/master/docs/",
+//          editUrl: "https://github.com/hwchase17/langchain/edit/master/docs/",
```

Comment:
why do we want to remove this again?

Reply:
This removes the “Edit this page” link at the end of pages. See this link on this [page](https://python.langchain.com/docs/modules/data_connection/retrievers/), for example.

Rationale: “Edit this page” link is misleading. For now, the documentation page changes are performed not just by some simple "edit_this_page" workflow but by full-pledged PRs. 

BTW This link is not working now.